mod/metrics/set/abstract/metric_answer/source_citation.rb

```diff
@@ -14,6 +14,10 @@
   end
```

Comment:
Do I understand correctly that we never get here for calculated answers (because they're not created)?